peps/pep-0621.rst

```diff
@@ -20,6 +20,9 @@ Post-History: 22-Jun-2020,
 Resolution: https://discuss.python.org/t/pep-621-round-3/5472/109
 
 
+.. canonical-pypa-spec:: :ref:`packaging:pyproject-toml-spec`
```

Comment:
I failed to find where `canonical-pypa-spec` is used. It is not rendered at https://peps.python.org/pep-0621/ (at least, I didn't see it).

What's the purpose of this line?

Reply:
It's not rendered at https://peps.python.org/pep-0621/ because you just deleted it in https://github.com/python/peps/pull/3564 :)


Here's how it used to look: https://web.archive.org/web/20230000000000*/https://peps.python.org/pep-0621/

![image](https://github.com/python/peps/assets/1324225/66dc0db9-9832-4c35-8b91-b5a99ca8e1fa)

It's a custom directive in the PEPs theme to create the banner:

https://github.com/python/peps/blob/99825461f8662458622f846f5d3b16727a0b2f6d/pep_sphinx_extensions/__init__.py#L75

Reply:
It's the "Attention" banner? Oh ok, now I get it. Well, I hope that the reference will be fixed in packaging soon.

Reply:
Yep. Or we can merge this PR to point to the new reference name:

https://pep-previews--3563.org.readthedocs.build/pep-0621/